main.go

```diff
@@ -60,8 +67,12 @@ func main() {
 			log.Panicf("Error when creating authz policy directory: %s", err.Error())
 		}
 	}
-
-	configDescriptor := collection.SchemasFor(collections.IstioRbacV1Alpha1Serviceroles, collections.IstioRbacV1Alpha1Clusterrbacconfigs, collections.IstioRbacV1Alpha1Servicerolebindings, collections.IstioSecurityV1Beta1Authorizationpolicies)
+	var configDescriptor collection.Schemas
+	if *authPolicyControllerOnlyMode {
+		configDescriptor = collection.SchemasFor(collections.IstioRbacV1Alpha1Serviceroles, collections.IstioRbacV1Alpha1Clusterrbacconfigs, collections.IstioRbacV1Alpha1Servicerolebindings, collections.IstioSecurityV1Beta1Authorizationpolicies)
```

Comment:
Should L72 and L74 be the same?

Reply:
No they are not supposed to be the same, thanks for catching that 👍🏽 . Updated the code to use only Beta version when running in the authPolicyControllerOnlyMode.